packages/leavittbook/src/components/leavitt-person-company-select-demo.ts

```diff
@@ -0,0 +1,158 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, query } from 'lit/decorators.js';
+import { h1, h2, h3, h5, p } from '@leavittsoftware/titanium-styles';
+import { LeavittPersonCompanySelectElement } from '@leavittsoftware/leavitt-elements/lib/leavitt-person-company-select';
+
+import StoryStyles from '../styles/story-styles';
+import api3UserService from '../services/api3-user-service.js';
+
+import '@leavittsoftware/titanium-button';
+import '@leavittsoftware/titanium-card';
+import '@leavittsoftware/leavitt-elements/lib/leavitt-person-company-select';
+import '../shared/code-block';
+import '../shared/story-header';
+import '@api-viewer/docs';
+
+@customElement('leavitt-person-company-select-demo')
+export class LeavittPersonCompanySelectDemoElement extends LitElement {
+  @query('leavitt-person-company-select[label="Default person or company"]') defaultSelect!: LeavittPersonCompanySelectElement;
+
+  static styles = [h1, h2, h3, h5, p, StoryStyles, css``];
+
+  focus() {
+    this.defaultSelect.focus();
+  }
+
+  reset() {
+    this.defaultSelect.reset();
+  }
+
+  #defaultStory() {
+    return html`
+      <div>
+        <h1>Default</h1>
+        <p>Default select styles</p>
+        <titanium-button raised @click=${this.focus}>Focus</titanium-button>
+        <titanium-button raised @click=${this.reset}>Reset</titanium-button>
+        <leavitt-person-company-select
+          label="Default person or company"
+          .apiService=${api3UserService}
+          validationMessage="Select a person or company"
+        ></leavitt-person-company-select>
+      </div>
+    `;
+  }
+
+  #requiredStory() {
+    return html`
+      <div>
+        <h1>Required</h1>
+        <p>Required select styles</p>
+        <leavitt-person-company-select
+          label="Leavitt person or company"
```

Comment:
Maybe just say "person or company" for all of the labels. It is close to/actually does overflow when the label is too long
![personcompany](https://user-images.githubusercontent.com/5438655/168177878-78fa6519-135e-4619-8912-3d8620786bd5.PNG)